docs/runbooks/LowKVMNodesCount.md

```diff
@@ -23,5 +23,5 @@ kubectl get nodes -o jsonpath='{.items[*].status.allocatable}' | grep devices.ku
 
 [Validate hardware virtualization support](https://kubevirt.io/user-guide/operations/installation/#validate-hardware-virtualization-support)
 
-If hardware virtualization is not available, then a [software emulation fallback](https://github.com/kubevirt/kubevirt/blob/master/docs/software-emulation.md) can be enabled.
-
+If hardware virtualization is not available, then a [software emulation fallback](https://github.com/kubevirt/kubevirt/blob/master/docs/software-emulation.md) can be enabled.  
+( **Note:** Software emulation is not supported in downstream )
```

Comment:
@andreyod  I think we should say OpenShift Virtualization instead of downstream. Do you agree?

Reply:
Sure